README.md

```diff
@@ -1,126 +1,64 @@
-  - Some commits are pushed directly to the master branch
-  - PRs are used inconsistently
-  - There is a 50% disparity in code contributions between teammates
-  - There is no README, or README is insufficient
-
-### JavaScript Style & Implementation
-
-* **4:** 
-  - There are no global variables aside from query selectors, `currentCover` and `savedCovers`.
-  - All functions strictly adhere to [SRP](http://knnthvu.weebly.com/srp-and-dry.html) and - with few exceptions - are around 10 lines of code or less
-  - All functions and variables are semantically and concisely named
-  - Uses logical operators instead of if/else statements where applicable
-  - Code contains no antipatterns
-* **3:** 
-  - Application uses event delegation correctly on dynamic elements
-  - Functions are [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and observe [SRP](http://knnthvu.weebly.com/srp-and-dry.html)
-  - Functions make use of parameters and arguments in order to be more dynamic
-* **2:** 
-  - Properly uses parameters and arguments when used
-  - Uses named functions instead of anonymous functions as event handlers in event listeners
-  - Correctly uses if/else statements to handle multiple paths of logic & error handling
-* **1:** 
-  - Crafts JS according to the [Turing JS style guide](https://github.com/turingschool-examples/javascript/tree/master/es5)
-
-
-### Functional Expectations
-
-Functionality is the least important piece of the rubric. It's included because it is another benchmark to gauge proficiency. However, you should not pursue functionality at the expense of code quality or the learning/growth of all team members.
-
-This means, we DO NOT want to see:
-- Code that completes iterations but is sloppy
-- One or both team members do not understand every single line of code
-- One or both team members skips the problem solving process (pseudocoding, talking out the problem, articulating, planning) in the pursuit of completing functionality
-
-* **4:** Applications completes one or more of the extensions without bugs
-* **3:** Application completes all the expectations of Iteration 4 without bugs
-* **2:** Application completes all the expectations of Iteration 3 without bugs
-* **1:** Application completes all the expectations of Iteration 2 without bugs
\ No newline at end of file
+# Title and Backcover
+###
+ROMCOM is a site which creates any bad paperback romance novel you want and can randomly populate its own covers as well, complete with images, cheesy titles, and corny taglines.
+
+ROMCOM was built using using Atom, git workflow, and HTML and CSS provided by Turing.
+
+## Project Status
+###
+The functionality to use the site is complete, however the site has not been tested thoroughly for user generated errors. 
+
+## Webite Screengrabs
+
+Generating random images:
+![image](https://user-images.githubusercontent.com/56229864/82391118-bb843200-99fd-11ea-904d-d58af18cf333.png)
+
+Saving and deleting saved images:

```

Comment:
We gotta figure out how to do line breaks on these images

Reply:
DONE!